script.js

```diff
@@ -612,7 +630,7 @@ window.GaiaAppIcon = (function(exports) {
   template.innerHTML =
     `<style>@import url(${stylesheet});</style>
      <div id='image-container'><div id="spinner"></div></div>
-     <div><div dir='auto' id='subtitle'></div></div>`;
+     <div id='title-container'><div dir='auto' id='subtitle'></div></div>`;
```

Comment:
nit: can we make sure we always use `"` in HTML attributes, even when quoted in JS strings?